test/remote/gateways/remote_plexo_test.rb

```diff
@@ -62,6 +62,15 @@ def test_successful_purchase_with_network_token
     assert_equal 'You have been mocked.', response.message
   end
 
+  def test_successful_inquire_with_payment_id
+    response = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success response
+    auth = response.authorization
+    inquire = @gateway.inquire(auth, @options)
+    assert_success inquire
+    assert_match auth, response.params['id']
+  end
+
```

Comment:
💬 Comment:

Maybe a test showing the difference (or the lack of) in the transaction structure between the original purchase and the inquire could be useful, so you can know how core is going to update the references and status.

Reply:
Yes I will include a test for it